fighthealthinsurance/ml/ml_policy_doc_helper.py

```
@@ -0,0 +1,967 @@
+                try:
+                    result = await cls._analyze_single_chunk(
+                        chunk=chunk,
+                        chunk_index=idx,
+                        user_question=user_question,
+                        plan_category=plan_category,
+                        remaining_counter=remaining_counter,
+                        remaining_count=remaining_count,
+                        total_chunks=total_chunks,
+                        progress_callback=progress_callback,
+                    )
+                except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)